src/Servers/IIS/AspNetCoreModuleV2/Symbols/Microsoft.AspNetCore.ANCMSymbols.csproj

```
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

```suggestion
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <IsPackable>$(PackNativeAssets)</IsPackable>
```
Should fix this for non-Windows builds